<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartForm</class>
 <widget class="QWidget" name="StartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgb(236, 232, 224);
}

QPushButton {
    background-color: rgb(255, 150, 0);

    color: bllack;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton:focus {
    background-color: rgb(255, 100, 0);

    color: bllack;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <item>
    <widget class="QPushButton" name="new_project_button">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Создать новый проект</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="open_project_button">
     <property name="text">
      <string>Открыть проект</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="settings_button">
     <property name="text">
      <string>Настройки</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
